amazon-kinesis-client/src/test/java/software/amazon/kinesis/retrieval/fanout/FanOutConsumerRegistrationTest.java

```diff
@@ -112,8 +112,8 @@ public void testConsumerAlreadyExistsMultipleCalls() throws Exception {
         assertThat(firstCall, equalTo(CONSUMER_ARN));
         assertThat(secondCall, equalTo(CONSUMER_ARN));
 
-        verify(client).describeStreamConsumer(eq(createDescribeStreamConsumerRequest(null)));
-        verify(client).describeStreamSummary(eq(createDescribeStreamSummaryRequest()));
+        verify(client, times(2)).describeStreamConsumer(any(DescribeStreamConsumerRequest.class));
+        verify(client).describeStreamSummary(any(DescribeStreamSummaryRequest.class));
```

Comment:
Same as above, This test is trying harder to recreate an already existing consumer by calling multiple times. Do we really care that we are verifying the expected increase in the internal API call (.describeStreamConsumer), or should we just care that we do not register that already existing consumer?

Note: If this API has strict TPS limits (Like DescribeStream), then perhaps we really do care anytime this API is invoked.